svvp.json

```diff
@@ -26,7 +26,6 @@
     "Profile Interrupt Test",
     "__Windows_Server_2022_reject_test_names",
     "UEFI GOP mode test",
-    "Verify IOMMU presence and configuration",
```

Comment:
I don't think it is a good idea to remove this while IOMMU is not always enabled.

Reply:
As for me `reject_test_names` is a list of unsupported tests at all. Yes, some platform will fail IOMMU presence tests but the simmilar situation with a lot of other tests. `Check SMBIOS *` can fail with BIOS due to bug in QEMU/Windows. `Crashdump Support Test` will fail when client does not have internet access. `Hardware Security Testability Interface Test` requied new OVMF with HSTI table. But in general with proper CLI, platform, OVMF, etc all of these tests will pass. 

SVVP testing is very specific and user should use the proper platform if want to pass all tests.

Reply:
The requirements of `Check SMBIOS *`, `Crashdump Support Test`, `Hardware Security Testability Interface Test` are not under the control of AutoHCK so it makes sense to require the user to fulfill them.

The situation is different for IOMMU. It is AutoHCK that controls the device to enable so AutoHCK should always enable IOMMU for SVVP if it is going to test `Verify IOMMU presence and configuration`.

Reply:
That is due to a bug in QEMU/Windows. AutoHCK will run SVVP tests properly if the bug gets fixed. However `Verify IOMMU presence and configuration` will not run unless AutoHCK enables IOMMU. In this sense, it is AutoHCK's responsibility to get it to work, and we should enable it when we satisfy the responsibility.

Isn't it a matter of adding `iommu` to `drivers` by the way?

Reply:
maybe better to add `viommu_state: true` to svvp config?

Reply:
That sounds good to me.

Reply:
in any case, we have a problem with re-adding devices. If `iommu` will be specified in `devices_list` and `viommu_state: true` is in the platform, AutoHCK adds iommu again.

Reply:
done